src/load.ts

```diff
@@ -82,6 +83,45 @@ export async function load(
     }
 }
 
+/**
+ * Loads the data from a CDN and returns an instance of AzureAppConfiguration.
+ * @param cdnEndpoint  The URL to the CDN.
+ * @param appConfigOptions  Optional parameters.
+ */
+export async function loadFromCdn(cdnEndpoint: URL | string, options?: AzureAppConfigurationOptions): Promise<AzureAppConfiguration>;
+
+export async function loadFromCdn(
+    cdnEndpoint: string | URL,
+    appConfigOptions?: AzureAppConfigurationOptions
+): Promise<AzureAppConfiguration> {
+    const emptyTokenCredential: TokenCredential = {
+        getToken: async () => ({ token: "", expiresOnTimestamp: 0 })
+    };
+
+    // the api version supports sas token authentication
+    const apiVersion = "2024-09-01-preview";
+    const policyName = "CdnRequestApiVersionPolicy";
+    const apiVersionPolicy: PipelinePolicy = {
+        name: policyName,
+        sendRequest: async (request: PipelineRequest, next: SendRequest) => {
+            const url = new URL(request.url);
+            url.searchParams.set("api-version", apiVersion);
+            request.url = url.toString();
+            return next(request);
+        },
+    };
+    if (appConfigOptions === undefined) {
+        appConfigOptions = { clientOptions: {}};
+    }
+    const policies = appConfigOptions.clientOptions?.additionalPolicies || [];

```

Comment:
We might want an additional 

```
if (appConfigOptions.clientOptions == null) {
    appConfigOptions.clientOptions = {}
}
```

Otherwise the policy wont be added if they provided an appConfigOptions with a null clientOptions